transformers/component_tests/queries/all_ilk_file_events_query_test.go

```diff
@@ -49,17 +50,24 @@ var _ = Describe("Ilk File Events Query", func() {
 		timestampOne = int(rand.Int31())
 		headerOne = createHeader(blockOne, timestampOne, headerRepo)
 		logOneId = test_data.CreateTestLog(headerOne.Id, db).ID
+		msgSender = test_data.Cat110Address()
```

Comment:
I wonder if we might want to use a different (potentially random) address here - the contract itself is unlikely to be the `msg.sender`, and I could see some tricky-to-debug test situations if the contract address and message sender IDs are the same

Reply:
Yeah.. makes sense. I think I spaced out on assigning the contract address as the msgSender.